metrics/share-of-voice.md

```diff
@@ -0,0 +1,56 @@
+# Share of Voice
+
+Can we communicate the value of our community as a comparison to something of value by our supporting business?
+
+## Description
+
+Ratio of conversation occurring by or about our community in comparison to some baseline. Also described as: Visibility, chatter, social media presence, top-of-funnel impact.
+
+## Objectives
+
+The metric supports the perception of continued value from community work in order to continue to secure funding. It highlights the value driven by community work by comparing it to something already valued by the organization. 
+
+## Implementation
+
+### Prerequisites
+
+Share of voice only works if you can agree to its implied assumptions ahead of time:
```

Comment:
I'm going through to make the language simpler and sharper. Disregard if I change the meaning in any of these edits.

```suggestion
Share of voice works if everyone can agree to its implied assumptions:
```